tests/framework-tests/preferences/js/PanelTestUtils.js

```diff
@@ -42,4 +45,10 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
         jqUnit.assertEquals(message, expectedState, checkbox.is(":checked"));
     };
 
+    fluid.tests.panels.utils.getTemplate = function (that) {
```

Comment:
You should probably put this in the same namespace as the component it is used by. Also, it would be good to have them adjacent to each other.